cartridges/bm_adyen/site-template-new/services.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<services xmlns="http://www.demandware.com/xml/impex/services/2014-09-26">
+    <service-credential service-credential-id="Adyen">
+        <url>https://pal-test.adyen.com/pal/adapter/httppost</url>
+        <user-id/>
+        <password/>
+    </service-credential>
+
+    <service-credential service-credential-id="Methods">
+        <url>https://test.adyen.com/hpp/directory.shtml</url>
+        <user-id/>
+        <password/>
+    </service-credential>
+
+    <service-credential service-credential-id="Send">
+        <url>https://pal-test.adyen.com/pal/servlet/Payment/v18/authorise</url>
```

Comment:
this is now applied https://github.com/Adyen/adyen-salesforce-commerce-cloud/pull/37/commits/523bee4b26de47c03fa786fcf61f08c3ff19dfbb#diff-5fea49d2670589116f78b4cf7ccfcca1R22